Source/Urho3D/Resource/ResourceCache.cpp

```diff
@@ -1086,7 +1086,7 @@ void ResourceCache::HandleBeginFrame(StringHash eventType, VariantMap& eventData
 
             VariantMap& eventData = GetEventDataMap();
             eventData[P_FILENAME] = fileWatchers_[i]->GetPath() + change.fileName_;
-            eventData[P_CHANGEDTPYE] = (int)change.type_;
+            eventData[P_CHANGETPYE] = (int)change.type_;

```

Comment:
typo?

Reply:
For consistency. In other places, we just use **change**

Reply:
I mean "TPYE"

Reply:
It's enum to int cast. If there is an issue, please be specific.